app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -28,30 +30,42 @@ IUSE=""
 
 DEPEND=">=dev-lang/go-1.3
 	app-arch/cpio
-	sys-fs/squashfs-tools"
+	sys-fs/squashfs-tools
+	dev-perl/Capture-Tiny"
 RDEPEND="!app-emulation/rocket"
 
-src_unpack() {
-	local cache="${S}/stage1/rootfs/usr_from_coreos/cache"
+src_configure() {
+	local myeconfargs=(
+		--with-stage1=host
```

Comment:
@alban --with-stage1=host is probably the right path but unless I missed something it pulls from `/` while we need to give it an arbitrary $ROOT, the build host is never has anything to do with the target we are building for.

Also `host` attempts to do thing like run `systemctl --version` which it must not do in our case as the build host's architecture may not match the targets.